modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -1242,6 +1242,7 @@ all-users-can-mention-each-other=All Users Can Mention Each Other
 all-users-of-x-are-automatically-members-of-the-x=All users of {0} are automatically members of the {1}. Administrators and Organization Owners of {0} are automatically assigned the roles {2} Administrator and {2} Owner respectively.
```

Comment:
@rbohl @lipusz just wanted to double check that you guys are good with the wording of all these new lang keys?

Reply:
Thanks for mentioning @BryanEngler. See my comments below :point_down: